Sources/FigmaExport/Subcommands/ExportIcons.swift

```diff
@@ -227,3 +229,18 @@ extension FigmaExportCommand {
         }
     }
 }
+
+private extension FigmaExportCommand.ExportIcons {
+    private func rewriteXMLFile(from source: URL, fileContents: FileContents) {
+        guard fileContents.isRTL else { return }
+        let sourceXML = try? XMLDocument(contentsOf: source, options: [.documentTidyXML])
+        try? sourceXML?.nodes(forXPath: "//vector").forEach { node in
+            guard let buttonElement = node as? XMLElement else { return }
+            let attribute: XMLNode = XMLNode.attribute(withName: "android:autoMirrored",
+                                                       stringValue: "\(fileContents.isRTL)") as? XMLNode
+            ?? XMLNode()
+            buttonElement.addAttribute(attribute)
+        }
+        try? FigmaExportCommand.fileWriter.write(xmlFile: sourceXML, directory: source)
```

Comment:
Do not suppress errors in this method. Use `try` and mark the method as `throws`.